tests/modelbench_tests/test_benchmark_grading.py

```diff
@@ -5,9 +5,9 @@
 import pytest
 
 from modelbench.benchmarks import BenchmarkScore, GeneralPurposeAiChatBenchmarkV1, Locale
-from modelbench.hazards import HazardScore, SafeCaeHazard, SafeCbrHazard, SafeHatHazard, Standards
+from modelbench.hazards import HazardScore, SafeHazardV1, Standards
```

Comment:
Do you know the original motivation for having one individual subclass for each hazard?

Reply:
No, I don't